src/views/Home.vue

```diff
@@ -8,7 +8,7 @@
     </p>
 
     <div class="alert alert-info">
-      <p>{{ $t("localSaveWarning") }}</p>
+      <p class="tinytext">{{ $t("localSaveWarning") }}</p>
```

Comment:
There are existing classes for this https://wet-boew.github.io/wet-boew-styleguide/design/sizing-en.html
Might also be an a11y issue if this is supposed to be important